Lib/asyncio/events.py

```diff
@@ -842,3 +843,13 @@ def set_child_watcher(watcher):
     _c_get_running_loop = get_running_loop
     _c_get_event_loop = get_event_loop
     _c__get_event_loop = _get_event_loop
+
+
+if hasattr(os, 'fork'):
+    def on_fork():
+        # Reset the loop and wakeupfd in the forked child process.
+        if _event_loop_policy is not None:
```

Comment:
Clever. If this is still `None` when `os.fork()` is called, there can't be any event loops, so there's nothing to reset.